src/main/java/at/hannibal2/skyhanni/features/mobs/SpawnTimers.kt

```diff
@@ -0,0 +1,40 @@
+package at.hannibal2.skyhanni.features.mobs
+
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.LorenzVec
+import at.hannibal2.skyhanni.utils.RenderUtils.drawDynamicText
+import at.hannibal2.skyhanni.utils.TimeUtils
+import net.minecraftforge.client.event.ClientChatReceivedEvent
+import net.minecraftforge.client.event.RenderWorldLastEvent
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+class SpawnTimers {
+    private var arachneSpawnTime = 0L
+
+    @SubscribeEvent
+    fun onRenderWorld(event: RenderWorldLastEvent) {
+        if (!LorenzUtils.inSkyBlock || arachneSpawnTime == 0L || LorenzUtils.skyBlockArea != "Arachne's Sanctuary") return;

```

Comment:
Add a toggle for the feature in the config and check if it's enabled in both events